src/Tribe/RSVP.php

```diff
@@ -2061,6 +2061,11 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 
 			// Attendee Meta
 			'attendee_meta' => $meta,
+
+			// Handle initial Attendee flags.
+			// @todo Make these live in future IAC work.
+			'is_subscribed' => false,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 6 space(s) between "'is_subscribed'" and double arrow, but found 1.

<sub>hash: 78c1507629bc016e97ad198980952ddf</sub>